.github/workflows/main.yml

```
@@ -0,0 +1,114 @@
+          echo "Desplegando Frontend..."
+          sudo rm -rf ${DEPLOY_PATH}/mi-app-frontend/browser_old
+          sudo mv ${DEPLOY_PATH}/mi-app-frontend/browser ${DEPLOY_PATH}/mi-app-frontend/browser_old || true
+          sudo mkdir -p ${DEPLOY_PATH}/mi-app-frontend/browser
+          sudo cp -r /home/runner/work/${{ github.repository }}/${{ github.repository }}/dist_frontend_temp/. ${DEPLOY_PATH}/mi-app-frontend/browser/
+          sudo chown -R www-data:www-data ${DEPLOY_PATH}/mi-app-frontend/browser
+          sudo chmod -R g+w ${DEPLOY_PATH}/mi-app-frontend/browser
+          sudo chmod g+s ${DEPLOY_PATH}/mi-app-frontend
+          echo "Reiniciando Nginx..."
+          sudo systemctl restart nginx
+          sudo nginx -t
+          echo "Despliegue completado."
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)